src/main/java/io/fusionauth/load/Foreman.java

```diff
@@ -114,9 +116,21 @@ public void initialize() {
       throw new IllegalStateException("Please provide a worker factory if you are going to specify a worker count");
     }
 
+    String directive = null;
+    String url = null;
+    String apiKey = null;
+    if (loadDefinition != null && loadDefinition.workerFactory != null) {
+      directive = loadDefinition.workerFactory.getString("directive", null);
+      url = loadDefinition.workerFactory.getString("url", null);
+      apiKey = loadDefinition.workerFactory.getString("apiKey", null);
+    }
+
     System.out.println("  --> Worker count:\t" + df.format(workerCount));
-    System.out.println("  --> Total iterations:\t" + df.format(workerCount * loopCount));
+    System.out.println("  --> Total iterations:\t" + df.format((long) workerCount * loopCount));
     System.out.println("  --> Worker factory:\t" + workerFactory.getClass().getCanonicalName());
+    System.out.println("  --> Worker directive:\t" + directive);
+    System.out.println("  --> Worker url:\t" + url);
+    System.out.println("  --> Target version:\t" + fetchVersion(url, apiKey));
```

Comment:
This is all ok in theory, but you are making a lot of assumptions here. 

The foreman shouldn't know anything about FusionAuth or the specific load test it is running. It is only starting workers, and handing them work. This framework is not supposed to be specific to FusionAuth, it is just a generic load test framework. 

Many of the workers are for FusionAuth things, so there is a `FusionAuthWorkerFactory` - it may be appropriate for that factory to know these types of things and report that to the `Foreman` or the `Forman` could just allow the `Factory` impl to print various things to the console as well during start up. 

We could create a new method off of the `WorkerFactory` that optionally returns various information that could be useful for the foreman to report on.